### DISCUSSION TITLE
feedback acknowledgment of notification to HA

### DISCUSSION THREAD

**Discussion by thefirstavi** [Blackymas/NSPanel_HA_Blueprint#1473] [Ideas]
Sorry if this is already suggested elsewhere.

My example is I have a simple toggle in HA which gives me whether the Recycling has been taken out. Ideally i would notify to NSPanel that recycling needs taking out today for collection. If the notification is ticked on the Panel, then i could change the status of the toggle. in HA.

**Comment by edwardtfn** [comment]
Sorry for the long delay in this reply.

When you send a notification to the panel, that notification will be sent back to HA via sensor **Notification text** and the switch **Notification unread** will turn on.
When someone see the notification and click on the close button (X), the **Notification unread** switch will turn off, but the notification still available to the panel.
When the notification is cleared (checkmark) then that sensor would be cleaned up.

You probably could use this for your automation.
